libs/cases/frontend/view/src/lib/components/create-case/create-case.component.ts

```diff
@@ -28,30 +34,38 @@ import { DateDropdownWrapperComponent } from './date-dropdown-accessor';
 	encapsulation: ViewEncapsulation.None,
 	changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class CreateCaseComponent {
+export class CreateCaseComponent implements OnInit {
     private readonly _browseFacade = inject(XdBrowseFacade);
     protected readonly _browseFacade2 = inject(XdBrowseFacadesService);
     protected readonly facilities = toSignal(this._browseFacade.getAllTimeseries());
 
+    facilityPlaceholder = signal('Select Facility');
+    typePlaceholder = signal('Select Type');
+    priorityPlaceholder = signal('Select Priority');
```

Comment:
Why do we need signals? Can't we just have them as text in the html file?

Reply:
U are right. Normally sth like this should *only* be part of the html file, since it doesnt influence logic.

Reply:
but the placeholder dynamically changes, as i have a method that copies the content of the select field to this signal

Reply:
Okay makes sense